### PR TITLE
Use the new elasticsearch client for version 5.0.

### PR DESCRIPTION
**Pull request by valentin-claras**
:warning: In Progress…
# Todo
Check everything still works the same

### CODE REVIEW

**Comment by fabdsp** [comment]
🍀

File: composer.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
     ],
     "require": {
         "php": ">=5.5",
-        "elasticsearch/elasticsearch": "^2.3"
+        "elasticsearch/elasticsearch": "5.0.x-dev"
```
Comment (b-viguier) [inline composer.json]: Branch 5.0 is now stable
https://github.com/elastic/elasticsearch-php#installation-via-composer

**Comment by omansour** [comment]
we can kik out the support for php 5.6

File: .travis.yml [hunk 2]
```diff
@@ -1,7 +1,6 @@
 language: php
 
 php:
-  - 5.5
   - 5.6
```
Comment (lucascourot) [inline .travis.yml]: @b-viguier this line should be removed too then?

**Comment by valentin-claras** [comment]
@lucascourot This is a bundle, not a full project, so there's no composer.lock commited ;)

**Pull request merged**